generate-zip.sh

```
@@ -0,0 +1,26 @@
+## Builds the app, then gets the App and the Test Runner and zips them
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)